classes/form/pointmarking.php

```diff
@@ -72,9 +74,18 @@ public function definition() {
                 $radioarray[] = $mform->createElement('radio', 'response', '', get_string('pass', 'observation'), 'Pass');
                 $radioarray[] = $mform->createElement('radio', 'response', '', get_string('fail', 'observation'), 'Fail');
                 $mform->addGroup($radioarray, 'radioar', get_string('passfailtype', 'observation'), array(' '), false);
-                $mform->setType('response', PARAM_TEXT); // Response will be Pass or Fail.
+                $mform->setType('response', PARAM_TEXT);
                 $mform->addRule('radioar', get_string('required', 'observation'), 'required', null, 'client');
                 break;
+            case \mod_observation\observation_manager::INPUT_EVIDENCE:
+                // Image upload here.
+                $maxbytes = 524288000; // This restricts the size of each individual file.
```

Comment:
This should likely be some sort of constant, or even better bonus points, converted to an activity setting potentially